dockerfiles/debian-buster-all/scripts/fetch_and_build_deb

```diff
@@ -175,4 +175,11 @@ mjobs="$(expr $procs + 1)"
 
 DEB_BUILD_OPTIONS="parallel=${mjobs}" debuild --prepend-path /usr/local/bin --preserve-envvar CONF_EXTRA_VERSION --preserve-envvar UNENCRYPTED_PACKAGE --preserve-envvar PACKAGE_ENCRYPTION_KEY -uc -us -B --lintian-opts --profile debian --allow-root
 
+## Rename all *.ddeb files into *.deb since we upload debug packages as deb
+for file in ../*.ddeb
+do
+  mv "$file" "${file%.ddeb}.deb"
```

Comment:
The file that you want to rename was in the parent directory. This command not only changes the extension of the file, but also moves it to current directory.

Is this really the expected behavior?